### ISSUE TITLE
非常需要香港登录beanfun

### ISSUE THREAD

**Issue by pickleman111** [kevin940726/BeanfunLogin#42]
是否有做香港登录beanfun
如果没有，我很樂意付費给予时间的补偿

有方便聯繫作者或者其他維護工作者的方式嗎

謝謝

**Comment by xnum** [comment]
有時間我會試試

**Comment by pickleman111** [comment]
謝謝回復

有qq 或者skype 方便加下聯繫方式嗎 我可以交流下  看下給更多輔助信息

**Comment by xnum** [comment]
似乎在台灣沒有辦法使用香港的BF網站，只會一直跳出請安裝元件

我這邊沒有用通訊軟體的習慣

**Comment by pickleman111** [comment]
对 香港的 需要安装那个 网页插件（IE支持）-》 这就是唯一个和 台服的不用支持，有个token是从他那个安装元件加载网页token加密生成的。 其他和台湾差不多

台湾和香港IP 通用香港BF网站 

如果需要我可以提供 IP和账号 也可提供其他需要的

**Comment by xnum** [comment]
最近工作忙，暫時沒有安排這項計畫
